deployment/dev/script/deploy.sh

```
@@ -1,18 +1,21 @@
 #!/bin/bash
 
 echo "========================================"
-echo "[DEPLOY] Blue/Green 무중단 배포 시작"
+echo "[DEV DEPLOY] 개발 서버 무중단 배포 실행"
 echo "========================================"
 
 cd "$(dirname "$0")/../blue-green"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)